test/unit/os/shell_spec.lua

```diff
@@ -27,16 +27,16 @@ describe('shell functions', function()
   setup(function()
     shell.event_init()
     -- os_system() can't work when the p_sh and p_shcf variables are unset
-    shell.p_sh = to_cstr('/bin/bash')
-    shell.p_shcf = to_cstr('-c')
+    cimported.p_sh = to_cstr('/bin/bash')
+    cimported.p_shcf = to_cstr('-c')
   end)
 
   teardown(function()
     shell.event_teardown()
```

Comment:
I left this and the `event_init()` call in `setup()` above just in case they were around for some SAN-related reason.